rpcd/playbooks/roles/rpc_support/tasks/create_default_flavors.yml

```diff
@@ -13,33 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install shade and its dependencies
-  pip:
-    name: "{{ item.name }}"
-    version: "{{ item.version }}"
-    extra_args: "--isolated"
-  with_items:
-    - { name: "shade", version: "1.16.0"}
-
+# TODO(evrardjp): Move back to the previous commit with os_ module in
+# a more recent branch, with compatible requirements AND playbook
+# will be targetting to good hosts directly.
 - name: Add default flavors
-  os_nova_flavor:
-    name: "{{ item.name }}"
-    vcpus: "{{ item.vcpus }}"
-    state: present
-    endpoint_type: "internal"
-    disk: "{{ item.disk }}"
-    ram: "{{ item.ram }}"
-    auth:
-      auth_url: "{{ keystone_service_internalurl }}"
-      username: "admin"
-      password: "{{ keystone_auth_admin_password }}"
-      project_name: "admin"
-      domain_name: "Default"
-    validate_certs: false
+  shell: |
+    openstack --os-cloud default flavor create --ram {{ item.RAM }} --disk {{ item.Disk }} --vcpus {{ item.VCPUs }} --public {{ item.Name }}
```

Comment:
I think you may need to source the openrc file, unless the clouds.yaml file is automatically discovered?